web_src/js/features/repo-issue.js

```diff
@@ -392,7 +392,8 @@ export async function handleReply($el) {
   const $textarea = form.find('textarea');
   let editor = getComboMarkdownEditor($textarea);
   if (!editor) {
-    editor = await initComboMarkdownEditor(form.find('.combo-markdown-editor'));
+    const editors = await initComboMarkdownEditor(form.find('.combo-markdown-editor'));
+    editor = editors[0];
```

Comment:
`initComboMarkdownEditor` returns a `ComboMarkdownEditor` class object, not a jQuery object.

I will propose new fixes.